tremor-cli/src/main.rs

```diff
@@ -22,6 +22,7 @@
     clippy::unnecessary_unwrap,
     clippy::pedantic
 )]
+#![cfg_attr(coverage, feature(no_coverage))]
```

Comment:
👀

Reply:
See: https://github.com/taiki-e/cargo-llvm-cov#exclude-function-from-coverage

Reply:
Damn, we can't use this on stable rust. :sob: